src/referencePackages/src/microsoft.dotnet.templatelocator/5.0.100-rc.1.20421.19/microsoft.dotnet.templatelocator.nuspec

```diff
@@ -0,0 +1,21 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>microsoft.dotnet.templateLocator</id>
```

Comment:
Locator is capitalized?  Is that intended?

Reply:
Yeah--the nupkg is a mess. It's `microsoft.dotnet.templatelocator.5.0.100-rc.1.20421.19.nupkg`, and the nuspec in that nupkg is `microsoft.dotnet.templatelocator.nuspec` but contains `<id>microsoft.dotnet.templateLocator</id>` (where this came from). And then the DLL is `Microsoft.DotNet.TemplateLocator.dll`. It's brought in by `PackageReference Include="Microsoft.DotNet.TemplateLocator"` in dotnet/templating so I don't think the nupkg/id casing will matter.